### PR TITLE
Update Excel (Vue) quick start

### PR DESCRIPTION
**Pull request by kbrandl**

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/excel-add-ins-get-started-vue.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-qs-excel-vue-20181018/docs/excel/excel-add-ins-get-started-vue.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-vue.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-qs-excel-vue-20181018/docs/includes/file-get-started-excel-vue.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-quickstart-vue.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-qs-excel-vue-20181018/docs/quickstarts/excel-quickstart-vue.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810181732250094-485%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=sKEatVYYX3JBpJ0hw%2Bu82Rd%2BpIpyEfP1lXJpU6VHI4A%3D&st=2018-10-18T17%3A28%3A42Z&se=2018-11-18T17%3A33%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/includes/file-get-started-excel-vue.md [hunk 1]
```diff
@@ -52,28 +54,27 @@ Each add-in requires a manifest file to define its settings and capabilities.
...
 ```json
 "dev": "webpack-dev-server --https --inline --progress --config build/webpack.dev.conf.js"
 ```
 
 ## Update the app
 
-1. In your code editor, open the manifest file (i.e., the file in the root directory of your app with a name ending in "manifest.xml"). Replace all occurrences of `https://localhost:3000` with `https://localhost:8080` and save the file.
+1. In your code editor, open the folder **My Office Add-in** that Yo Office created at the root of your Vue project. In that folder, you'll see the manifest file that defines the settings for your add-in: **my-office-add-in-manifest.xml**.
+
+2. Open the manifest file, replace all occurrences of `https://localhost:3000` with `https://localhost:8080`, and save the file.
```
Comment (AlexJerabek) [inline docs/includes/file-get-started-excel-vue.md]: General question: why is this step necessary? Are we trying to show how to change where the add-in looks for the service? Or is this a Vue specific step?

File: docs/includes/file-get-started-excel-vue.md [hunk 2]
```diff
@@ -52,28 +54,27 @@ Each add-in requires a manifest file to define its settings and capabilities.
     - **What do you want to name your add-in?:** `My Office Add-in`
     - **Which Office client application would you like to support?:** `Excel`
 
-    After you complete the wizard, a manifest file and resource file are available for you to build your project.
-
     ![Yeoman generator](../images/yo-office.png)
     
-    > [!NOTE]
-    > If you're prompted to overwrite **package.json**, answer **No** (do not overwrite).
+    After you complete the wizard, the generator will create the manifest file.
```
Comment (AlexJerabek) [inline docs/includes/file-get-started-excel-vue.md]: Optional nit: "generator will create" -> "generator creates"

File: docs/includes/file-get-started-excel-vue.md (hunk 1 above)
Comment (kbrandl) [reply docs/includes/file-get-started-excel-vue.md]: It's Vue-specific -- by default, a locally-hosted Vue app runs on port 8080.

File: docs/includes/file-get-started-excel-vue.md (hunk 2 above)
Comment (kbrandl) [reply docs/includes/file-get-started-excel-vue.md]: Fixed; thanks.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/excel-add-ins-get-started-vue.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-qs-excel-vue-20181018/docs/excel/excel-add-ins-get-started-vue.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-vue.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-qs-excel-vue-20181018/docs/includes/file-get-started-excel-vue.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-quickstart-vue.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-qs-excel-vue-20181018/docs/quickstarts/excel-quickstart-vue.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810181905088049-485%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Qj8IrxNLYQ2KPKZ%2F2Q9xp%2Bn%2BaKfQ4uAMaqqx0CfKWFI%3D&st=2018-10-18T19%3A01%3A25Z&se=2018-11-18T19%3A06%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**